File: Python/Program/Baju.py
from Aksesoris import Aksesoris

class Baju(Aksesoris):
    def __init__(self, id, nama, harga, stok, jenis, bahan, warna, untuk, size, merk):
        super().__init__(id, nama, harga, stok, jenis, bahan, warna)
        self.__untukHewan = untuk
        self.__size = size
        self.__merk = merk

    def setUntukHewan(self, untuk):
        self.__untukHewan = untuk
    def getUntukHewan(self):
        return self.__untukHewan
    
    def setSize(self, size):
        self.__size = size
    def getSize(self):
        return self.__size
    
    def setMerk(self, merk):
        self.__merk = merk
    def getMerk(self):
        return self.__merk

File: Java/Program/Baju.java
class Baju extends Aksesoris{
    private String untukHewan;
    private String size;
    private String merk;
    
    Baju(String id, String nama, int harga, int stok){
        super(id, nama, harga, stok);
        this.untukHewan = "";
        this.size = "";
        this.merk = "";
    }

    public void setUntukHewan(String untuk){
        this.untukHewan = untuk;
    }
    public String getUntukHewan(){
        return this.untukHewan;
    }

    public void setSize(String size){
        this.size = size;
    }
    public String getSize(){
        return this.size;
    }

    public void setMerk(String merk){
        this.merk = merk;
    }
    public String getMerk(){
        return this.merk;
    }
}